modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -38,6 +38,11 @@ Object Store v2 uses TLS for secure transport.
 Data at rest is stored using FIPS 140-2-compliant encryption standards.
 If you require a higher level of security, encrypt sensitive data before writing it to the object store.
 
+=== Synchronize Access to Object Store v2
+
+Because CloudHub doesn't support distributed locking, errors or data discrepancies can occur if any components that use Object Store v2 attempt to update a key simultaneously.
```

Comment:
OSv2 can only be used in CloudHub. Seems like an irrelevant detail for the customer. Can we frame this better?
Put the focus on how the customer should deal with the lack of this capability.

Reply:
I like how simple and clear the previous version was. Can we have a modified version of that:
"Object Store v2 does not coordinate key updates as it does not include a distributed locking mechanism for keys. If multiple Mule components attempt to update a key simultaneously, errors or data discrepancies can occur."

Reply:
Hi,  It is incorrect to say that OSv2  *does not* include distributed locking as even if it wants to provide distributed locking, it cannot. Distributed locking is a mule interface that needs to be implemented by CH. It sends a wrong message to our customers/field and support if we say "Object Store v2 does not coordinate key updates as it does not include a distributed locking mechanism for keys."